### PR TITLE
NONE: remove more data from logs

### PR DESCRIPTION
**Pull request by bgvozdev**
**What's in this PR?**
Removing Auth header from logs

**Why**
Dislike

### CODE REVIEW

File: src/config/logger.ts [hunk 1]
```diff
@@ -105,6 +105,20 @@ const censorUrl = (url) => {
...
+	}
+
+	const ret = { ...headers };
+	if (ret["Authorization"] || ret["authorization"]) {
+		delete ret["Authorization"];
+		delete ret["authorization"];
+		ret["Authorization"] = "CENSORED";
+	}
+	return ret;
+};
```
Comment (mboudreau) [inline src/config/logger.ts:120]: if you're going to do this, wouldn't it make more sense to "normalize" all headers to be lowercase?

File: src/config/logger.ts [hunk 2]
```diff
@@ -105,6 +105,19 @@ const censorUrl = (url) => {
 	return url;
 };
 
+const headersSerializer = (headers) => {
+	if (!headers) {
+		return headers;
+	}
+
+	const ret = mapKeys({ ...headers }, (_, key) => key.toLowerCase());
```
Comment (mboudreau) [inline src/config/logger.ts:113]: why are you recreating an object from the headers?  Headers should already be an object :P

**Review by mboudreau — APPROVED** [review]

File: src/config/logger.ts (hunk 2 above)
Comment (bgvozdev) [reply src/config/logger.ts:113]: Immutability, they say ;)

Comment (bgvozdev) [reply src/config/logger.ts:113]: Ah wait a sec, lodash is immtable 🤦  I'll fix that, good catch @mboudreau

**Review by mboudreau — APPROVED** [review]

**Pull request merged**